docs/website/docs/plus/ecosystem/snowflake_plus.md

```diff
@@ -124,6 +124,56 @@ Snowflake Plus supports all the data types from the standard Snowflake destinati
 | `binary` | `binary` |
 | `json` | `string` |
 
+
+## Syncing Snowflake-managed Iceberg tables to Snowflake Open Catalog
+
+To enable querying of Snowflake-managed Iceberg tables by third-party engines (e.g., Apache Spark) via an external catalog (Snowflake Open Catalog), use the `catalog_sync` configuration option. This setting specifies a [catalog integration](https://docs.snowflake.com/en/user-guide/tables-iceberg#catalog-integration) that syncs Iceberg tables to the external catalog.
+
+### Setup
+
+1. [Create an external catalog in Snowflake Open Catalog](https://other-docs.snowflake.com/en/opencatalog/create-catalog).
```

Comment:
```suggestion
1. Create an external catalog in Snowflake Open Catalog according to their [instructions](https://other-docs.snowflake.com/en/opencatalog/create-catalog).
```

Reply:
Not sure about this suggestion: 
1. "their" creates ambiguity about who they are. The reader can infer that this is Snowflake, but I think this ambiguity is unnecessary.
2. the original version is better since the linked text clearly describes the destination.

Reply:
It's not good practice to put an entire sentence as the URL. So maybe just Snowflake Open Catalog should be the link. But I'll leave you to decide